test/unit/vl-select.test.html

```diff
@@ -238,11 +238,22 @@
       });
 
       test('wanneer het vl-select element volledig klaar is resolved de ready functie', (done) => {
-        const element1 = fixture('vl-select-fixture');
-        element1.dress();
-        element1.ready().then(() => done());
+        const element = fixture('vl-select-fixture');
+        element.dress();
+        element.ready().then(() => done());
       });
 
+      test('wanneer het select element een slot attribuut bevat, zal het na de dress het slot attribuut kopiëren naar het wrapper element', (done) => {
```

Comment:
ik mis hier een beetje 'waarom' het dat zou doen ...
ook de beschrijving van de feature of de PR zegt niet veel over wat er nu juist moet gebeuren of welk probleem opgelost wordt.

Reply:
en moet ge hier dan eender welk ander attribuut ook niet gaan kopieren naar de wrapper? nu ist slot da ge nodig hebt maar morgen is het foo en overmorgen bar?

Reply:
@pascalriquier ik heb wat context toegevoegd aan de beschrijving van de PR. Ik zou (voorlopig) alleen het slot attribuut verplaatsen, ik zie op dit moment geen use case om ook nog andere attributen te verplaatsen naar de container. Wanneer dit later alsnog nodig zou blijken, kunnen we het uiteraard dan voorzien.